packages/puppeteer-core/src/cdp/Browser.ts

```diff
@@ -483,4 +485,16 @@ export class CdpBrowser extends BrowserBase {
   override isNetworkEnabled(): boolean {
     return this.#networkEnabled;
   }
+
+  // Current Bluetooth simulation is implemented on the browser level, and not tight to
+  // the specific target. `cdpBluetoothEmulationManager` is a singleton until CDP
+  // implementation is adjusted.
+  _cdpBluetoothEmulationManager(): CdpBluetoothEmulationManager {
+    if (!this.#cdpBluetoothEmulationManager) {
+      this.#cdpBluetoothEmulationManager = new CdpBluetoothEmulationManager(
+        this.target(),
```

Comment:
we should not base it on the deprecated target API? let's either pass the Page object or the primary client.

Reply:
done